src/schema/v2/fair.ts

```diff
@@ -139,14 +139,18 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
       exhibitionPeriod: {
         type: GraphQLString,
         description: "A formatted description of the start to end dates",
-        resolve: ({ start_at, end_at }) => dateRange(start_at, end_at, "UTC"),
-      },
-      shortExhibitionPeriod: {
-        type: GraphQLString,
-        description:
-          "A formatted description of the start to end dates with abbreviated months",
-        resolve: ({ start_at, end_at }) =>
-          dateRange(start_at, end_at, "UTC", true),
+        args: {
+          format: {
+            type: ExhibitionPeriodFormatType,
+            description:
+              "SHORT or LONG, SHORT returns a shorter exhbition period description with months abbreviated, defaults to LONG",
```

Comment:
**suggestion**: Instead of implicitly including default behavior, the GraphQL SDL supports an explicit `defaultValue` option. With an explicit default value in place, you might also be able to make the argument `non null`:

```typescript
{
  exhibitionPeriod: {
    type: GraphQLString,
    description: "A formatted description of the start to end dates",
    args: {
      format: {
        type: new GraphQLNonNull(ExhibitionPeriodFormatType),
        description:  "Formatting option to apply to exhibition period",
        defaultValue: ExhibitionPeriodFormatType.LONG.value
      }
    }
  }
}
```